buildpipeline/pipeline.json

```diff
@@ -110,7 +110,9 @@
           "Parameters": {
             "PB_DistroRid": "win7-x64",
             "PB_TargetArchitecture": "x64",
-            "PB_PortableBuild": "true"
+            "PB_PortableBuild": "true",
+            "PB_PublishRidAgnosticPackages": "true",
```

Comment:
NOTE: This question is mostly for my understanding at our high-level direction.

If we just named these parameters the same as the MSBuild properties, then we wouldn't need to pass them into `/p` on the command-line.  They would just automatically be set for the entire build.

Why prefix these variables with `PB_` only to trim the `PB_` off and pass the value through?  This seems like an extra step that can easily cause type-o errors and/or missing setting it when the build is split across multiple steps.